### ISSUE TITLE
Some config defaults are not set to publicly accessible URLs

### ISSUE THREAD

**Issue by raoulmillais** [tes/bosco#241]
Browser-sync default URL:

https://github.com/tes/bosco/blob/60543640d4196813aeb78af63a93fae948ae9734/config/options.json#L58

Default docker registry:

https://github.com/tes/bosco/blob/03594cc3299739603685bafe01e5a0af2f37e1f8/src/RunListHelper.js#L38

**Comment by cliftonc** [comment]
This was done on purpose (early days) to avoid people at Tes having to set these properties, e.g. the defaults work for Tes folks, but you can over-ride them if you're not at Tes.

Happy to replace them with something more generic, but we then need to get every dev at Tes to update their local bosco config.

**Comment by raoulmillais** [comment]
Yup I see why it was done.  I raised it because the docs suggest bosco is designed for use outside TES and this seemed surprising in that light.  I guess it really depends on how widely bosco is  used outside TES and whether we care about less friction for 3rd parties or for TES folks
